README.md

```diff
@@ -510,7 +510,9 @@ ort {
 }
 ```
 
-While the specified schema already needs to exist, the _scanner_ will itself create a table called `scan_results` and
+The database needs to exist. If the schema is defined and is not the default postgress _public_, it need to be previously created and grant usage for the specific database user. Left schema entry empty will always default to _public_.
```

Comment:
Second sentence:

    If the schema is set to something else than the default of "public", it needs to exist and be accessible by the configured username.